pom.xml

```diff
@@ -22,7 +22,6 @@
         <module>modules/GraphvizLayout</module>
         <module>modules/GraphStreaming</module>
         <module>modules/DesktopStreaming</module>
-        <module>modules/StreamingAPI</module>
```

Comment:
Oh, I didn't notice you are not the original developer of the plugin. 
In that case, you should work with the original developer branch: https://github.com/totetmatt/gephi-plugins/tree/multimode-network and then create a pull request there with your changes. Then the original developer should merge your changes and create another pull request here.